crates/jstz_cli/src/sandbox/mod.rs

```diff
@@ -10,20 +10,19 @@ mod daemon;
 use crate::config::Config;
 
 #[derive(Subcommand)]
-pub enum Command {
+pub enum SandboxCommand {
```

Comment:
Any reason for the rename?

Reply:
it was conflicting with process::Command, but I've later decided to move it to daemon.rs, so I guess it could be changed back to Command.